core/domain/feedback_services.py

```diff
@@ -115,8 +118,16 @@ def create_message(
             thread.subject = updated_subject
     thread.put()
 
+    if (user_services.is_user_registered(author_id) and len(text) > 0 and
```

Comment:
Btw, just checking: what happens if a new suggestion is created? (You can make one by logging in as a learner and editing a card of an exploration.) Does an email get sent then? Is it clear that it's a suggestion?